cli/spinner.ts

```diff
@@ -136,13 +136,16 @@ export class Spinner {
     // Updates the spinner after the given interval.
     const updateFrame = () => {
       const color = this.#color ?? "";
-      Deno.stdout.writeSync(LINE_CLEAR);
       const frame = encoder.encode(
         noColor
           ? this.#spinner[i] + " " + this.message
           : color + this.#spinner[i] + COLOR_RESET + " " + this.message,
       );
-      Deno.stdout.writeSync(frame);
+      // call writeSync once to reduce flickering
+      const writeData = new Uint8Array(LINE_CLEAR.length + frame.length);
+      writeData.set(LINE_CLEAR);
+      writeData.set(frame, LINE_CLEAR.length);
+      Deno.stdout.writeSync(writeData);
```

Comment:
An alternative using [`concat()`](https://jsr.io/@std/bytes@1.0.0-rc.3/doc/concat/~/concat):
```suggestion
      Deno.stdout.writeSync(concat([LINE_CLEAR, frame]));
```

Reply:
Oh, I missed that this was in a separate package. I'm not sure it's worth causing another package load to occur when importing this module.